daofund-score/src/main/java/com/iconbet/score/daofund/DaoFund.java

```diff
@@ -0,0 +1,303 @@
+	    for index in range(len(array_db) - 1):
+	        if array_db[index] == target:
+	            array_db[index] = _out
+	            array_db.pop()
+	            return True
+	    return False
+***/
+
+	@External(readonly = true)
+	public String name() {
+		/***
+		:return: name of the Score
+		***/
+		
+		return TAG;
+	}
+	
+	@External
+	public void add_admin(Address _admin) {
+		Address sender = Context.getCaller();
+		Address owner = Context.getOrigin();
+		if (!sender.equals(owner)) {
+			Context.revert(TAG + ": Only admins can set new admins.");
+		}
+		
+		if (!containsInArrayDb(_admin, this.admins)) {
+			this.admins.add(_admin);
+			AdminAdded(_admin);
+			
+		}else {
+			Context.revert(TAG + ":  "+ _admin +" is already on admin list.");
+		}
+	}
+	
+	@External
+	public void remove_admin(Address _admin) {
+		Address sender = Context.getCaller();
+		Address owner = Context.getOrigin();
+		if (!sender.equals(owner)) {
+			Context.revert(TAG + ": Only admins can remove admins.");
+		}
+		
+		if (_admin.equals(owner)) {
+			Context.revert(TAG + ": Owner address cannot be removed from the admins list.");
+		}
+		
+		if (containsInArrayDb(_admin, this.admins)) {
+			remove_array_item(_admin, this.admins);
+			AdminRemoved(_admin);
+		}else {
+			Context.revert(TAG + ":  "+ _admin +" not in Admins List");
+		}
+	}
+	
+	@External(readonly = true)
+	public List<Address> get_admins() {
+		
+		List<Address> addressList = new ArrayList<>();
+
+		for (int i=0; i< this.admins.size(); i++) {
+			addressList.add(this.admins.get(i));
+		}
+		return addressList;
+	}
+	
+	/***
+	 * Add fund to the daoFund wallet
+	 ***/
+	@External
+	@Payable
+	public void add_fund() {
+		
+	}
+		 
+	@External
+	public void withdraw_fund( Address _address, BigInteger _amount, String _memo) {
+		Address sender = Context.getCaller();
+		
+		if (!containsInArrayDb(sender, this.admins)) {
+			Context.revert(TAG + ": Only admins can run this method.");
+		}
+		//  _available_amount = self.icx.get_balance(self.address)
+		BigInteger _available_amount = BigInteger.ZERO;
+		if ( _available_amount.compareTo(_amount) == -1 ) {
+			Context.revert(TAG + ": Not Enough balance. Available Balance =" + _available_amount.toString());
+		}
+		
+		try {
+			
+			BigInteger _count = this.withdraw_count.get();
+			BigInteger _withdraw_count = _count.add(BigInteger.ONE);
+			
+			BigInteger now = BigInteger.valueOf(Context.getBlockTimestamp());		
+			BigInteger day = BigInteger.ZERO;
+			day = day.add(now.divide(X_6));
+			
+			this.withdraw_count.set(_withdraw_count);
+			getWithdrawRecord("withdraw_amount").set(_withdraw_count.intValue(),_amount.toString() ) ;
+			getWithdrawRecord("withdraw_address").set(_withdraw_count.intValue(),_address.toString() ) ;
+			getWithdrawRecord("withdraw_memo").set(_withdraw_count.intValue(), _memo ) ;
+			getWithdrawRecord("withdraw_timestamp").set(_withdraw_count.intValue(), day.toString() ) ;
+			// self.icx.transfer(_address, _amount)
```

Comment:
i do not see the java  Context.transfer call here

Reply:
done.